docs/ironic-endpoint-keepalived-configuration.md

```diff
@@ -0,0 +1,123 @@
+# Maintain Ironic Endpoint with Keepalived
+
+The motivation behind maintaining Ironic Endpoint with Keepalived is to ensure
+that the Ironic Endpoint IP is also passed onto the target cluster control
+plane. This also guarantees that once pivoting is done and the management
+cluster is taken down, target cluster controlplane can re-claim the ironic
+endpoint IP through keepalived. The end goal is to make ironic endpoint
+reachable in the target cluster.
+
+To this end, we have restructed the `baremetal-operator/deploy` folder
+structure to better reflect the kustomization workflow since now we have
+different deployment scenarios i.e. operator-with-ironic or operator-with-
+ironic-and-keepalived. The following directory structure will help you to
+visualize this in more detail.
```

Comment:
The following directory tree visualizes the new structure.

Reply:
Done